src/java.base/share/classes/java/lang/AbstractStringBuilder.java

```diff
@@ -1821,4 +1821,115 @@ private final void appendChars(CharSequence s, int off, int end) {
         }
         count += end - off;
     }
+
+    /**
+     * Appends {@code count} copies of the character {@code c} to this sequence.
+     * <p>
+     * The length of this sequence increases by {@code count}.
+     *
+     * @param c      character to append
+     * @param count  number of times to copy
+     *
+     * @return  a reference to this object.
+     *
+     * @since 21
+     * @throws IllegalArgumentException  if {@code count} is less than zero
+     * @throws StringIndexOutOfBoundsException  if the result overflows the buffer
+     */
+    public AbstractStringBuilder repeat(char c, int count) {
+        if (count < 0) {
+            throw new IllegalArgumentException("count is less than zero: " + count);
+        }
+        if (count == 0) {
+            return this;
+        }
+        ensureCapacityInternal(this.count + count);
+        if (isLatin1() && StringLatin1.canEncode(c)) {
+            int index = this.count;
+            while (count-- != 0) {
+                value[index++] = (byte) c;
+            }
+            this.count = index;
+        } else {
+            if (isLatin1()) {
+                inflate();
+            }
+            int index = this.count;
+            while (count-- != 0) {
+                StringUTF16.putCharSB(value, index++, c);
+            }
+            this.count = index;
+        }
+        return this;
+    }
+
+    private AbstractStringBuilder repeatNull(int count) {
+        if (count < 0) {
```

Comment:
This could be implemented as `repeat("null", count)`. Its not likely to be performance sensitive and be easier to maintain.

Reply:
Just being consistent with existing code.

Reply:
Better to add less code to maintain later.